src/app/components/forms/contact-form/contact-form.component.scss

```diff
@@ -0,0 +1,10 @@
+.contact-button-container {
+  margin: 0 auto;
```

Comment:
Properties should be ordered width, padding-bottom, margin

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)